src/appleseed/renderer/modeling/frame/frame.cpp

```diff
@@ -100,26 +96,28 @@ UniqueID Frame::get_class_uid()
 struct Frame::Impl
 {
     // Parameters.
-    size_t                  m_frame_width;
-    size_t                  m_frame_height;
-    size_t                  m_tile_width;
-    size_t                  m_tile_height;
-    string                  m_filter_name;
-    float                   m_filter_radius;
-    unique_ptr<Filter2f>    m_filter;
-    AABB2u                  m_crop_window;
-    bool                    m_render_stamp_enabled;
-    string                  m_render_stamp_format;
-    DenoisingMode           m_denoising_mode;
-    bool                    m_save_extra_aovs;
+    size_t                          m_frame_width;
+    size_t                          m_frame_height;
+    size_t                          m_tile_width;
+    size_t                          m_tile_height;
+    string                          m_filter_name;
+    float                           m_filter_radius;
```

Comment:
Align name

Reply:
Everything is aligned, as far as I can see...

Reply:
Weird..
![2018-06-11-08 59 36](https://user-images.githubusercontent.com/4656466/41216779-098e2c3e-6d56-11e8-8f4a-1ac854724303.png)

Reply:
Your browser is using a proportional font, for some reason...